scss/mixins/_close.scss

```
@@ -38,7 +38,7 @@
       background-image: str-replace($close-icon, str-replace(#{'fill=\'' + $close-color + '\''}, "#", "%23"), str-replace(#{'fill=\'' + color-yiq($color) + '\''}, "#", "%23")); // SEB specific
       opacity: .5;
     }
-    &:not(:disabled):not(.disabled):hover::after, &:not(:disabled):not(.disabled):focus::after {
+    &:not(:disabled):not(.disabled)::after {
```

`disabled` here is repeated twice
```suggestion
    &:not(.disabled)::after {
```

---

@yousifalraheem have you hade any issues with the disabled state? If you look closely it's actually not a typo or duplication, the reason behind applying both is that there's a difference between `.disabled` and `:disabled`:)